slugid/slugid.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under the Mozilla Public Licence 2.0.
+# https://www.mozilla.org/en-US/MPL/2.0
+
+import uuid
+import base64
+
+def encode(uuid_):
+    """
+    Returns the given uuid as a 22 character slug. This can be a regular v4
+    slug or a "nice" slug.
+    """
+    bytes = uuid_.bytes
+    slug = base64.urlsafe_b64encode(str(bytes))[:22] # Drop '==' padding
```

Comment:
Do we want the first 22 characters or remove the last two? If it is the second option, I rather prefer the `[:-2]` idiom.